### DISCUSSION TITLE
Debugging Existing Job

### DISCUSSION THREAD

**Discussion by rcronin** [codefori/vscode-ibmi#2198] [General]
I saw this in the launch.json generated for the IBM i Debugger extension:

<img width="456" alt="image" src="https://github.com/user-attachments/assets/f7257692-0581-4659-b06f-475c9680b120">

Has anybody been able to get the Attach to running job working?

**Comment by rcronin** [comment]
@worksofliam - wanted to tag you to see if you had answer on this? Thanks!

**Reply by worksofliam** [reply]
We use the launch API inside of the extension. Do a search of our codebase for 'startdebugging' and you should find where we are calling the VS Code API with the launch.json object.
